assignment_template/Analysis.py

```diff
@@ -27,11 +28,45 @@ def load_data(self) -> None:
         print(self.config['figure_title'])
 
     def compute_analysis(self) -> Any:
-        pass
+        '''Analyze previously-loaded data.
+
+        This function compute the mean of pokemons' base experience 
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        analysis_output : Float
+
+        '''
+        return np.mean(self.dataset["base_experience"])
```

Comment:
Have you seen what the data frame looks like from what I sent you? Want to do more than just 1 mean? Add max, min, mean to height and weight

Reply:
and count number of types

Reply:
I could add more stats.
Also,example data would look like this- so I think the code should be okay, but I'll add test
```
{'base_experience': 64,
 'height': 7,
 'id': 1,
 'species': 'bulbasaur',
 'type1': 'grass',
 'type2': 'poison',
 'weight': 69}
```